Fix get package details issue

Trying to run the `package-stats-command.ts` I'm getting an error

```bash
Argument 'field' on InputObject 'PackageFileOrder' has an invalid value (UPDATED_AT). Expected type 'PackageFileOrderField'.
Elapsed time so far: 0h 0m 1s
2026-04-03T01:12:05.178Z info: Fetching maven packages for organization: itops
2026-04-03T01:12:05.178Z info: Fetching package page 1 with cursor: initial
2026-04-03T01:12:05.282Z error: Error processing organization itops: Operation failed after 3 attempts: Request failed due to following response errors:
Argument 'field' on InputObject 'PackageFileOrder' has an invalid value (UPDATED_AT). Expected type 'PackageFileOrderField'.
Stack trace: GraphqlResponseError: Request failed due to following response errors:
Argument 'field' on InputObject 'PackageFileOrder' has an invalid value (UPDATED_AT). Expected type 'PackageFileOrderField'.
at file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/node_modules/@octokit/graphql/dist-bundle/index.js:89:13
at async OctokitClient.getOrgPackageDetails (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:760:30)
at async processingState.successCount (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:8432:26)
at async withRetry (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:3785:28)
at async Object.processOrgPackageStats [as processOrg] (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:8426:5)
at async executeForOrg (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:6531:9)
at async executeCommand (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:6457:24)
at async runPackageStats (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:8340:20)
at async Command.<anonymous> (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:8583:5)
file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:6469
throw new Error(Stopping processing due to error (use --continue-on-error to continue));
^
Error: Stopping processing due to error (use --continue-on-error to continue)
at executeCommand (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:6469:19)
at async runPackageStats (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:8340:20)
at async Command.<anonymous> (file:///home/runner/.local/share/gh/extensions/gh-repo-stats-plus/dist/index.js:8583:5)
Node.js v25.9.0
Error: Process completed with exit code 1.
```

I think we didn't copy over something on the `queries.ts` from the original source of this command which comes from

--- a/src/queries.ts
+++ b/src/queries.ts
@@ -335,7 +335,7 @@ export const ORG_PACKAGE_DETAILS_QUERY = `
             downloadsTotalCount
           }
           latestVersion {
-            files(first: 100, orderBy: {field: UPDATED_AT, direction: DESC}) {
+            files(last: 100, orderBy: {field: CREATED_AT, direction: ASC}) {
               nodes {
                 name
                 size

